packages/wdio-mocha-framework/src/index.js

```diff
@@ -166,16 +166,17 @@ class MochaAdapter {
         if (params.payload) {
             message.title = params.payload.title
             message.parent = params.payload.parent ? params.payload.parent.title : null
-
+            
             /**
              * get title for hooks in root suite
              */
             if (message.parent === '' && params.payload.parent && params.payload.parent.suites) {
                 message.parent = params.payload.parent.suites[0].title
             }
-
+            
```

Comment:
```suggestion

```

Reply:
I added the file property to the unit tests.

Reply:
@ablok please also remove the spaces as suggested above

Reply:
@christian-bromann Wow, I really thought it already did that after I committed your suggested change. Well, shoud be fixed now.